CRM/Financial/Form/Export.php

```diff
@@ -31,7 +31,7 @@ class CRM_Financial_Form_Export extends CRM_Core_Form {
 
   /**
    * Financial batch ids.
-   * @var array
+   * @var string (comma-separated array)
```

Comment:
This change appears to be incorrect. The variable is initialized as an array, not a string.

Reply:
I think this is actually a string, based on how the property is used:

 - Whenever the propery is used it's preceeded by an `explode(',', $_GET['batch_id'])`, which indicates a string is expected.
 - In `CRM_Batch_BAO_Batch::getBatchNames`, `$this->_batchIds` is passed in as the first agument which then gets fed directly into a `WHERE id IN(...)` database query.

So I think this is a string, which gets exploded whenever it needs to be used as a true PHP array.